src/v/redpanda/application.cc

```diff
@@ -680,6 +715,12 @@ void application::wire_up_redpanda_services() {
           std::ref(arch_configs))
           .get();
         arch_configs.stop().get();
+
+        construct_service(
+          _upload_controller,
```

Comment:
I think we'd better stick to consistent naming for subsystems and use these names for everything that is related to these subsystems. In this case I would expect "archival" to be part of the name somewhere.

Reply:
good point, fixed